include/RAJA/policy/openmp/policy.hpp

```diff
@@ -76,12 +118,7 @@ struct omp_for_nowait_exec
 };
 
 template <unsigned int N>
-struct omp_for_static : make_policy_pattern_launch_platform_t<Policy::openmp,
-                                                              Pattern::forall,
-                                                              Launch::undefined,
-                                                              Platform::host,
-                                                              omp::For,
-                                                              omp::Static<N>> {
+struct omp_for_static : omp_for_schedule<omp::Static<N>> {
```

Comment:
Could this be an alias to avoid generating another class?

Reply:
You mean the omp::Static part?  Will was working on a PR to clean up some of this. We briefly discussed at the RAJA meeting earlier this week. I'll defer to the OpenMP experts.